### PR TITLE
Completing Test Recipe for Command Runner

### PR DESCRIPTION
**Pull request by MariusNi**
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-osconfig/blob/main/CONTRIBUTING.md).
- [X] I added unit-tests to validate my changes. All unit tests are passing.
- [X] I have merged the latest `main` branch prior to this PR submission.
- [X] I submitted this PR against the `main` branch.

### CODE REVIEW

File: src/modules/test/recipes/CommandRunnerTests.json [hunk 1]
```diff
@@ -1,40 +1,124 @@
...
+    "Desired": false,
+    "ExpectedResult": 0,
+    "WaitSeconds": 0
+  },
+  {
+    "ComponentName": "CommandRunner",
+    "ObjectName": "commandArguments",
+    "Desired": true,
+    "Payload": "{\"commandId\":\"TestCommandOne\",\"arguments\":\"echo Test\",\"timeout\":0,\"singleLineTextResult\":true,\"action\":3}",
+    "PayloadSizeBytes": 0,
```
Comment (AhmedBM) [inline src/modules/test/recipes/CommandRunnerTests.json:14]: not necessary, default=0 which, uses the size of the payload defined above

**Review by AhmedBM — APPROVED** [review]
Looks good, however the `PayloadSizeBytes` are not needed unless you want to explicitly set something other than the payload size as defined in the `Payload` itself

File: src/modules/test/recipes/CommandRunnerTests.json (hunk 1 above)
Comment (MariusNi) [reply src/modules/test/recipes/CommandRunnerTests.json:14]: Good! I will remove then

**Comment by MariusNi** [comment]
Good! Will remove these

---
In reply to: [1082871971](https://github.com/Azure/azure-osconfig/pull/352#issuecomment-1082871971) [](http://example.com/codeflow?ancestors=1082871971,1082871971)

**Review by AhmedBM — APPROVED** [review]
👌

**Pull request merged**